openreview/conference/templates/seniorAreaChairWebfield.js

```diff
@@ -14,11 +18,97 @@ var main = function() {
   }
 
   renderHeader();
+
+  getRootGroups()
+  .then(loadData)
+  .then(formatData)
+  .then(renderTableAndTasks)
+  .fail(function() {
+    Webfield.ui.errorMessage();
+  });
 };
 
 var renderHeader = function() {
   Webfield.ui.setup('#group-container', CONFERENCE_ID);
   Webfield.ui.header(HEADER.title, HEADER.instructions);
+
+  var loadingMessage = '<p class="empty-message">No assigned Area Chairs.</p>';
+  var tabsList = [
+    {
+      heading: 'Assigned Area Chairs',
+      id: 'assigned-areachairs',
+      content: loadingMessage,
+      extraClasses: 'horizontal-scroll',
+      active: true
+    },
+    {
+      heading: 'Senior Area Chair Tasks',
+      id: 'senior-areachair-tasks',
+      content: loadingMessage
+    }
+  ];
+
+  Webfield.ui.tabPanel(tabsList);
+};
+
+var getRootGroups = function() {
+  // Get paper groups or assignments
+
+  return $.Deferred().resolve([]);
```

Comment:
Is this needed for future features or could you just use loadData?

Reply:
Yes, I was following the code of the AC console, I may need to load the edges here.